lib/src/components/board/board.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:viam_sdk/src/resource/base.dart';
+
+import '../../gen/common/v1/common.pb.dart' as common;
+import '../../robot/client.dart';
+
+class BoardStatus {
+  final Map<String, int> analogs;
+  final Map<String, int> digitalInterrupts;
+
+  BoardStatus(this.analogs, this.digitalInterrupts);
```

Comment:
Should we make this a `cost` initializer since all attributes are final?

Reply:
yes we should. Good catch, thanks!